src/vit-servicing-station/vit-servicing-station-lib/migrations/postgres/2022-11-29-134506_add_proposals_extra_fields/up.sql

```diff
@@ -0,0 +1,41 @@
+ALTER TABLE proposals ADD proposal_extra_fields TEXT;
```

Comment:
Using `TEXT` here so vit-ss code remains compatible with Postgres and SQLite (I guess SQLite supports JSON type only as an extension).